### PR TITLE
Include requirements-*.txt, pyx and pxd, exclude c file in MANIFEST.in

### PR DESCRIPTION
**Pull request by chineking**
## What do these changes do?

Include requirements-extra.txt and requirements-dev.txt in MANIFEST.in. Global include pyx and pxd, exclude c file in MANIFEST.in.

## Related issue number

Fix file not found mentioned in #152

### CODE REVIEW

File: requirements-extra.txt [hunk 1]
```diff
@@ -5,6 +5,5 @@ psutil>=4.0.0
 lz4>=1.0.0
 protobuf>=3.6
 gevent>=1.3.0
-etcd-gevent>=0.1.0
```
Comment (wjsi) [inline requirements-extra.txt]: Is there any error messages about etcd-gevent so we can fix?

Comment (chineking) [reply requirements-extra.txt]: I collect the feedback from users that the dependency like `geventhttpclient` is hard to install.

**Comment by wjsi** [comment]
Maybe we need to give separate guidances both in docs and README on building from sources or prepare a development environment.

**Comment by chineking** [comment]
> Maybe we need to give separate guidances both in docs and README on building from sources or preparing a development environment.

Agree, an issue can be created to mention this.

File: requirements-extra.txt (hunk 1 above)
Comment (wjsi) [reply requirements-extra.txt]: Source packages are always hard to install. Okay, forget that.

**Comment by wjsi** [comment]
etcd-gevent need to be installed in .travis.yml, as now it is removed from extras.

**Pull request merged**